README.md

```diff
@@ -4,6 +4,78 @@ Use this project to get started with Twilio Video's Android SDK by [following al
 
  [ ![Download](https://api.bintray.com/packages/twilio/releases/conversations-android/images/download.svg) ](https://bintray.com/twilio/releases/conversations-android/_latestVersion)
 
+## Up and Running
+
+1) Create a Twilio Video [Configuration Profile](https://www.twilio.com/user/account/video/profiles). If you haven't used Twilio before, welcome! You'll need to [Sign up for a Twilio account](https://www.twilio.com/try-twilio).
+
+2) Download this project and open it in Android Studio.
+
+3) Get an access token [Generate an Access Token](https://www.twilio.com/user/account/video/dev-tools/testing-tools). Pick your identity (such as Bob). Leave this web page open, because you'll use it as the other side of the video chat.
+
+4) Paste the access token into ConversationActivity.java
+
+5) Run your app on an Android device
+
+6) On the same web page where you generated the token, scroll down the bottom, put in the username that you generated the access token for, and click Create Conversation. Your video conversation should start immediately! 
+
+## What is this project?
+
+This quick start will help you get video chat integrated directly into your Android applications using Twilio's Video Conversations SDK. 
+
+Inside this quick start project, you will find an Activity that contains all of the functionality necessary to show two video streams on one screen - one video stream for your phone's video camera, and one for a remote video stream.
+
+You'll see how to set up key classes like TwilioAccessManager, ConversationsClient, LocalMedia, Conversation, and CameraCapturer. All of these Twilio classes have related listeners with implementations in ConversationActivity.java.
+
+You can also see how Android's runtime permissions are implemented for access to the camera and the microphone on Android devices running version 6.0 (Marshmallow) or higher inside the onCreate() method in the ConversationActivity class. It's important to note that you shouldn't initialize the Twilio client until the user grants access to the camera and microphone.
+
+## Prerequisites
+
+This project is a native Android application written in Java. You will need an up-to-date version of Android Studio (at the time of this writing, 2.1.0), to build the application. You will also need version 23.0.2 of the Android build tools, which you can download through Android Studio if you do not have them installed.
+
+You should use an Android device for testing - the minimum supported Android version is 4.0.3.
```

Comment:
This is not a hard requirement anymore. The latest conversations SDK supports the x86 emulators (32 and 64 bit)